### PR TITLE
HPCC-17422 DOCS:BackupNode component added

### PR DESCRIPTION
**Pull request by g-pan**
Fix HPCC-17422 DOCS:BackupNode component added
Update documentation to include the BackupNode
component added to the ConfigManager
Added enhancements to most (config manager) images. 

@JamesDeFabia please review
@Michael-Gardner please review

(User friendly) Output from test build can be viewed at  http://10.240.32.243/view/Docs/job/DocDev05/13/CE=ubuntu-14.04-amd64-1/
 

Signed-off-by: G-Pan <greg.panagiotatos@lexisnexis.com>

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browers
  - [ ] The component(s) render as expected

## Testing:

Test succeeded and output can be viewed at : 
http://10.240.32.243/view/Docs/job/DocDev05/13/CE=ubuntu-14.04-amd64-1/
(Using Config Manager pdf)

### CODE REVIEW

**Comment by richardkchapman** [comment]
Needs rebasing for a clean merge

**Comment by Michael-Gardner** [comment]
@g-pan Looks awesome and the info is correct

File: docs/UsingConfigManager/UsingConfigManager.xml [hunk 1]
```diff
@@ -706,6 +706,67 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
...
+
+            <listitem>
+              <para>Right click on the <emphasis
+              role="bold">computer</emphasis> column and choose <emphasis
+              role="bold">Add Instances</emphasis>...<graphic
+              fileref="images/CM-img22.jpg" vendor="configmgrSS" /></para>
+            </listitem>
+
+            <listitem>
+              <para>Select the computer for the backupnode component, or press
```
Comment (JamesDeFabia) [inline docs/UsingConfigManager/UsingConfigManager.xml]: Which server would I choose in a large system? ThorMaster? One of the support nodes?

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
comment inline

File: docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 2]
```diff
@@ -786,8 +786,9 @@
 
           <para>Backupnode is a tool that is packaged with HPCC. Backupnode
           allows you to backup Thor nodes on demand or in a script. You can
-          also use backupnode regularly in a crontab. You would always want to
-          run it on the Thor master of that cluster.</para>
+          also use backupnode regularly in a crontab or by adding a backupnode
+          component with configuration manager to your environment. You would
```
Comment (JamesDeFabia) [inline docs/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml]: Configuration Manager

File: docs/UsingConfigManager/UsingConfigManager.xml [hunk 3]
```diff
@@ -706,6 +706,69 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
       environment XML file.</para>
 
       <sect2 role="brk">
+        <title>Backup Node</title>
+
+        <para>Backup Node allows you to back up Thor clusters at regular
+        intervals. The Backup Node component is a way to allow administrators
+        to manage the backup node process without using a cron job. </para>
```
Comment (JamesDeFabia) [inline docs/UsingConfigManager/UsingConfigManager.xml]: backupnode process

File: docs/UsingConfigManager/UsingConfigManager.xml [hunk 4]
```diff
@@ -706,6 +706,69 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
...
+
+        <para>To enable back up node, you add the backupnode component, choose
```
Comment (JamesDeFabia) [inline docs/UsingConfigManager/UsingConfigManager.xml]: consisteency:  backupnode

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
Just some minor edits (see inline)

**Comment by g-pan** [comment]
Revised, Rebased, and Resubmit.

File: docs/UsingConfigManager/UsingConfigManager.xml (hunk 4 above)
Comment (JamesDeFabia) [inline docs/UsingConfigManager/UsingConfigManager.xml]: To enable back up node
s/b
To enable backupnode

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
One comment

**Review by JamesDeFabia — APPROVED** [review]

File: docs/UsingConfigManager/UsingConfigManager.xml [hunk 5]
```diff
@@ -706,6 +706,70 @@ sudo -u hpcc cp /etc/HPCCSystems/source/NewEnvironment.xml /etc/HPCCSystems/envi
       environment XML file.</para>
 
       <sect2 role="brk">
+        <title>Backupnode</title>
+
+        <para>Backupnode allows you to back up Thor clusters at regular
+        intervals. The Backupnode component is a way to allow administrators
+        to manage the backupnode process without using a cron job.</para>
+
+        <para>To enable this functionality, you add the backupnode component,
```
Comment (JamesDeFabia) [inline docs/UsingConfigManager/UsingConfigManager.xml]: To enable this functionality,
would be better to say:
To configure scheduled Thor node backups,

**Review by JamesDeFabia — CHANGES_REQUESTED** [review]
one minor comment

**Review by JamesDeFabia — APPROVED** [review]
Looks good to merge

**Pull request merged**